lib/data/symbols.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+// TODO: Update
```

Comment:
TODO? Do we need these symbols?

Reply:
They are used to provide constants, i.e. `SYMBOLS.BTC_USD` here:
```js
const market = {
  symbol: SYMBOLS.BTC_USD,
  tf: TIME_FRAMES.ONE_HOUR
}
```

Do you think we should remove them? Or maybe optionally load the list when the file is required() and fallback to the hardcoded list if there is no network? Would be simple.

Reply:
RIght I see, we do something similar for golang. Nah lets keep it like this, im a bit worried the network loading version would be a bit too 'magical' and might cause for some nasty error debugging